testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1460,3 +1462,15 @@
   raise 'File injection failed' unless return_code.zero?
   node.run('salt-call --local --file-root=/usr/share/susemanager/salt --module-dirs=/usr/share/susemanager/salt/ --log-level=info --retcode-passthrough state.apply ' + state)
 end
+
+When(/^I set correct product for "(proxy|branch server)"$/) do |product|
+  if product.include? 'proxy'
+    prod = 'SUSE-Manager-Proxy'
```

Comment:
Please don't use 'product' as variable, it is issue to mix with '$product'

Reply:
Also, review if for Uyuni, we use a different name like Uyuni-Proxy

Reply:
@srbarrios Very good catch, thanks! I didn't realize that, but yes, you're completely right.

Reply:
I'm not the author of this code.

If you would like such changes, please contact the original authors, and let's do it in separate PRs.

This is only about aligning the code in every branch.

Reply:
> I'm not the author of this code.
> 
> If you would like such changes, please contact the original authors, and let's do it in separate PRs.
> 
> This is only about aligning the code in every branch.

@Bischoff Do you want to merge something wrong? In Uyuni we need to distinguish with Head. This doesn't apply to spacewalk. So please don't merge it, to fix this wrong code in a future PR.

Reply:
We can also fix it as part of this PR, yes, if it is short.

It is just that I would like to focus on synchronizing the branches.

If we find errors at this occasion, fine, but maybe it is cleaner and simpler to do it in separate PRs. That's all.

Reply:
If we do in a separate PR better, but then let's not port any line at that section.

Reply:
@srbarrios I would like to do it in separate PR. TBH I am afraid we can jump into a lot of unexpected consequences as we entering area, where are differences between uyni, head and SUMA itself. I am author of this code, so I volunteer to fix it. But my suggestion is not to do it today on Friday before Christmas. I can create card for it so this won't be forgotten.

Reply:
> If we do in a separate PR better, but then let's not port any line at that section.

OK. I will not port anything then. Same for the other one above.

@lkotek i create a card for you in January?

Reply:
I just realized this code is used in QAM only - no uyuni then, it will become used in 4.2 where we have SUSE Manager only.

But... as we might want to start using QAM test suite at some point for milestones as well, better have so bullet-proof code.